health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/AdvancedMapCharts/HeatMap/HeatMapChart.js

```
@@ -0,0 +1,391 @@
+import React, { useCallback, useContext, useEffect, useMemo, useRef, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { LatLngBounds } from "leaflet";
+import { Loader, RemoveableTag } from "@egovernments/digit-ui-react-components";
+import Map from "./Map";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)